app/containers/Avatar/index.tsx

```diff
@@ -74,7 +83,7 @@ const AvatarContainer = ({
 	};
 
 	useEffect(() => {
-		if (!avatarETag) {
+		if (!avatarETag || avatarETag !== previousAvatarEtag.current) {
```

Comment:
Can you explain the lifecycle of these effects?
I didn't get how we have a query subscribe, but yet we need two effects to trigger it again.